kitsune/wiki/templates/wiki/edit.html

```diff
@@ -22,6 +22,14 @@
 
     <article id="edit-document" {% if locked %}class="locked"{% endif %} data-slug="{{ document.slug }}">
       <h1>{{ _('<em>Editing</em> {title}')|fe(title=document.title) }}</h1>
+      {% if document.needs_change %}
+        <p>
+          {% trans comment=document.needs_change_comment %}
+          <label>Needs Change:</label> {{comment}} 
```

Comment:
This line needs indented, since it is nested in the {% trans %} block. After that, I think this is ready to go.